src/perl/Permabit/BlockDevice/VDO/LVMVDO.pm

```diff
@@ -1,9 +1,9 @@
 ##
-# Perl object that represents a VDO Volume managed by LVM
+# Base Perl object that represents a VDO Volume managed by LVM
 #
```

Comment:
Did github auto-detect this file motion? It's weird to think of LVMManaged.pm turning into the base class.